lib/manager/components/version/FeedVersionSpanChart.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+
+import React, { Component } from 'react'
+import { Table } from 'react-bootstrap'
+import moment from 'moment'
+
+import type { FeedVersion } from '../../../types'
+
+type Props = {
+  activeVersion: FeedVersion,
+  comparedVersion: FeedVersion,
+  comparedVersionIndex: number
+}
+
+const dateFormat = 'MMM. DD, YYYY'
+
+/** Renders the validity span of one feed using the given props. */
+const FeedSpan = ({
+  endLabel,
+  relativeLength,
+  relativeOffset,
+  startLabel,
+  widthPixels,
+  ...props
```

Comment:
I like to avoid using `...props` because it can lead to not qualifying component props or carelessly passing through many props. And it is unclear what props are even being used in the div anyways.

Reply:
Updated in https://github.com/ibi-group/datatools-ui/pull/528/commits/08c881ccaf552b503dddbb05ada677e37e3a9272.